datasimulator/file_handling.py

```diff
@@ -0,0 +1,20 @@
+from .utils import attempt  # same as datasimulator.utils
+from cdislogging import get_logger
+logger = get_logger("data-simulator", log_level="info")
+
+
+def write_node_to_file(outfile, node):
+    outfile.write(f"{node.name}\t{node.category}\n")
```

Comment:
i don't think this needs to be split into a separate function, it's a one-liner and is only used once. too many layers make the code harder to read